### PR TITLE
Zapier OAuth2 Docs update

### PR DESCRIPTION
**Pull request by kuzmany**
https://github.com/mautic/mautic-zapier/pull/18

### CODE REVIEW

File: en/plugins/zapier.md [hunk 1]
```diff
@@ -4,12 +4,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
...
+Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
 
 ## Installation
 
-There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic. This Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
+There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic.
+
+### Oauth2 Authentication for Zapier 3.x
+
+New version of app improve authentication by Oauth2 protocol.
```
Comment (RCheesley) [inline en/plugins/zapier.md]: This new version of the app improves authentication by implementing the Oauth2 protocol

File: en/plugins/zapier.md [hunk 2]
```diff
@@ -4,12 +4,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
...
+
+1. Open the admin menu
+2. Go to Mautic's global configuration > API Settings
+3. Set __API enabled?__ to __Yes__.
+4. Save the configuration.
+5. Go to Mautic's global configuration > API Credentials > __Oauth2__
+6. Create new __Oauth2___ application with redirect URI: __tba__
+7. Copy Public Key and Secret Key of your created app
+
+![How to enable API](https://user-images.githubusercontent.com/462477/74520415-cc616b00-4f17-11ea-8dbe-f6dd8a0a6cfc.png)
```
Comment (RCheesley) [inline en/plugins/zapier.md]: How to enable the API

File: en/plugins/zapier.md [hunk 3]
```diff
@@ -4,12 +4,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
...
+
+![New Oauth2 application](https://user-images.githubusercontent.com/462477/74520342-a1771700-4f17-11ea-866f-5be1c895f0f9.png)
+
+ 
+### Basic Authentication app for Zapier 2.x 
+ 
+ If you're using Zapier 2.x, this Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
```
Comment (RCheesley) [inline en/plugins/zapier.md]: If you are using Zapier 2.x, Basic Authentication is used to access the Mautic API. This is disabled by default, so you will need to enable it first.

File: en/plugins/zapier.md [hunk 4]
```diff
@@ -63,12 +81,9 @@ At this point choose which Trigger or Action you need. Each trigger will get you
 
 ### 3. Authorize Mautic instance
 
-Once you choose to use Mautic integration you'll need to authorize your Mautic to it. Mautic use basic auth as mentioned earlier. So all you need is a Mautic user credentials and URL of where your Mautic lives. It's recommended to create a new user for Zapier which will have some advantages:
-
-1. Giving a third party app credentials to your Mautic is a security risk. If something happens you simply delete this special Zapier user and your admin user will be safe.
-2. You will see what contacts were created by Zapier simply by looking at the created by user.
+Once you choose to use Mautic integration you'll need to authorize your Mautic to it.
```
Comment (RCheesley) [inline en/plugins/zapier.md]: Once you choose to enable the integration, you will need to authorize Mautic to communicate with Zapier.

**Review by RCheesley — CHANGES_REQUESTED** [review]
I've made a few minor suggestions - please also take a look at https://github.com/mautic/documentation/blob/master/CONTRIBUTING.md for our guidelines on documentation and give me a shout if anything is unclear!

**Comment by kuzmany** [comment]
@RCheesley I see. Thanks

**Comment by kuzmany** [comment]
@escopecz How we handle with basic auth info. Should remove it, or just notice about changes from basic auth to oauth. Or let setup also for 2.x (basic auth) and 3.x (oauth2) separately

**Comment by escopecz** [comment]
I guess we can remove it. The question is whether we want to wait with the Zapier Oauth2 PR till we fix the refresh token expiration or add to the docs to increase the refresh token lifetime to some big number.

**Comment by kuzmany** [comment]
@escopecz that refresh token expiration should be updated on every access token generate. That means, app stop working just If there is no usage during that expiration (default 14 days). Should we add just note to docs?

**Comment by escopecz** [comment]
@kuzmany you don't sound confident :) You said a customer of yours already runs this version. There are no problems with timeouts? Is their expiration value default at the Mautic API configuration?

A note in the docs would be great.

**Comment by kuzmany** [comment]
@escopecz It was bug, we didn't store updated refresh_token. But at the moment looks good. I did few tests

**Pull request closed without merging**